app/discover/[from]/page.tsx

```diff
@@ -1,37 +1,49 @@
 import { getTranslations } from "next-intl/server";
 import { dehydrate, HydrationBoundary } from "@tanstack/react-query";
-import { Suspense } from "react";
-import { DiscoverEventsList } from "./discover-events-list";
+import { redirect, RedirectType } from "next/navigation";
+import { DiscoverEventsList } from "../discover-events-list";
 import { getQueryClient } from "@/lib/get-query-client";
 import { ScreenContainer } from "@/components/layout/ScreenContainer";
 import { eventsList } from "@/lib/queries/events-list";
 import { EventsListLayout } from "@/components/layout/events-list-layout";
 
-export const revalidate = 60;
-export const dynamicParams = true;
+export const revalidate = 10;
```

Comment:
revalidate 10 seems very low considering the query takes 7 sec on staging
I would put back 60sec or even more